index.html

```diff
@@ -0,0 +1,229 @@
+                <h5 class="font-weight-bold">Lorem, ipsum dolor sit amet consectetur adipisicing elit.</h5>
+                <h6 class="font-weight-bold text-uppercase ">Posted on <span style="color: #eb5750">august 18 2019</span> by <span style="color: #eb5750">Admin</span></h6>
+                <p class="text-left pt-2 text-secondary" >Maecenas maximus, justo rhoncus egestas auctor, elit lorem finibus tellus, cursus sagittis orci nibh et leo. Praesent ut tincidunt dui. Etiam venenatis dolor vitae eleifend cursus. Aliquam erat volutpat. Maecenas sapien nisl, accumsan eget risus quis, tempus condimentum nulla. Sed ac lectus imperdiet, ultricies enim porta, fringilla metus.</p>
+            </div>
+        
+    </section>
+
+    <!--FOOTER-->
+
+    <footer id="footer-section" class="bg-dark text-light w-100">
+        <div class="container">
+            <div class="row">
+              <div class="col">
+                <b>REVERA</b>
+                <p class="text-secondary font-weight-light"></br>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed non eros mauris. Sed at aliquam lectus, quis facilisis diam. Suspendisse in aliquam diam. Etiam consectetur fringilla venenatis. Praesent ligula purus, faucibus in tellus ut, elementum auctor ipsum. Pellentesque mattis metus non maximus tempus. Sed euismod rutrum tellus in mollis.</p>
+              </div>
+
+              <div class="col">
+                <b>META</b>
+
+                <ul class="text-secondary list-group list-group-flush">
+                </br>
+                  <li class="list-group-item">Cras justo odio</li>
+                  <li class="list-group-item">Dapibus ac facilisis in</li>
+                  <li class="list-group-item">Morbi leo risus</li>
+                  <li class="list-group-item">Porta ac consectetur ac</li>
+                  <li class="list-group-item">Vestibulum at eros</li>
+                </ul>
+                
+              </div>
+
+              <div class="col">
+                <b>ARCHIVES</b>
+
+                <ul class="text-secondary list-group list-group-flush">
+                </br>
+                  <li class="list-group-item">August 2017</li>
+                  
+                </ul>
+              </div>
+              <div class="col">
+                <b>RECENT POSTS</b>
+
+                <ul class="text-secondary list-group list-group-flush">
+                </br>
+                  <li class="list-group-item">Etiam mauris tortor, pharetra lobortis in, pharetra in diam</li>
+                  <li class="list-group-item">Suspendisse et libero nunc. Etiam mauris tortor, pharetra </li>
+                  <li class="list-group-item">Etiam mauris tortor, quis lobortis in, pharetra in diam</li>
+                  <li class="list-group-item">Nulla facilisi. Nulla egestas at nunc at volutpat.</li>
+                  
+                </ul>
+              </div>
+            </div>
+          
```

Comment:
For the copyright bar, you can replace your `<footer>` on line 178 with a `<section>` element then you can use `<footer>` under `</section>` and add `<div class = "container"><div class="row">Copyright © 2018 revera </div></div>`